src/app/guildmanager/guildmanager.module.ts

```diff
@@ -10,13 +10,14 @@ import { SidenavComponent } from './components/sidenav/sidenav.component';
 import { MainContentComponent } from './components/maincontent/maincontent.component';
 import { RouterModule, Routes } from '@angular/router';
 import { GuildTableComponent } from './components/guildtable/guildtable.component';
+import { UserManagerAppComponent } from '../usermanager/usermanagerapp.component';
 
 const routes: Routes = [
-  { path: '', component: GuildManagerAppComponent,
+  { path: 'guilds', component: GuildManagerAppComponent,
     children: [
       { path: '', component: MainContentComponent }
-    ] },
-  { path: '**', redirectTo: '' }
```

Comment:
The use case is a "catch all" for any undefined routes.  I was having some problems triggered the correct routes because the tutorial I followed used "nested routing modules" aka the main module has one router, and the feature modules have their own routers as well...Tbh it confused me...I should clean it up or learn exactly how it's working.